test/algorithms/buffer/test_buffer.hpp

```diff
@@ -25,7 +25,8 @@
 #include <iomanip>
 
 #include <boost/foreach.hpp>
-#include "geometry_test_common.hpp"
+#include <geometry_test_common.hpp>
+#include <expectation_limits.hpp>
```

Comment:
Where is this file located? I can't find it.

Reply:
Oops - it wasn't there. But now it is. Thank you.